gulpfile.babel.js

```diff
@@ -181,6 +181,10 @@ gulp.task('lint:json', () => {
       quiet: false,
       fail: true
     }))
+    .on('error', (err) => {
```

Comment:
Shouldn't this be a separate PR?
I don't really care but its a comment you often raise :)